src/components/app/data/services/academies.js

```diff
@@ -11,14 +12,10 @@ export async function fetchAcademies(enterpriseUUID, options = {}) {
   const url = `${getConfig().ENTERPRISE_CATALOG_API_BASE_URL}/api/v1/academies?${queryParams.toString()}`;
 
   try {
-    const response = await getAuthenticatedHttpClient().get(url);
-    const data = camelCaseObject(response.data);
-    return data.results ?? [];
+    const { results } = await fetchPaginatedData(url);
+    return results;
   } catch (error) {
-    const errorResponseStatusCode = getErrorResponseStatusCode(error);
-    if (errorResponseStatusCode === 404) {
```

Comment:
[inform] API does not return a 404 when there are no results; no need to check 404 status code explicitly.